src/game/Accounts/AccountMgr.cpp

```diff
@@ -55,9 +55,21 @@ AccountOpResult AccountMgr::CreateAccount(std::string username, std::string pass
     const char* s_hex = srp.GetSalt().AsHexStr();
     const char* v_hex = srp.GetVerifier().AsHexStr();
 
+    // Get the current time point
+    std::chrono::system_clock::time_point now = std::chrono::system_clock::now();
+
+    // Convert the time point to a time_t for C-style date and time functions
+    std::time_t now_c = std::chrono::system_clock::to_time_t(now);
+
+    // Convert the time_t to a struct tm for formatted output
+    std::tm* now_tm = std::localtime(&now_c);
+
+    char buffer[80];
+    std::strftime(buffer, 80, "%Y-%m-%d %H:%M:%S", now_tm);
+
     bool update_sv = LoginDatabase.PExecute(
-        "INSERT INTO account(username,v,s,joindate) VALUES('%s','%s','%s',NOW())",
-            username.c_str(), v_hex, s_hex);
+        "INSERT INTO account(username,v,s,joindate) VALUES('%s','%s','%s','%s')",
+            username.c_str(), v_hex, s_hex, buffer);
```

Comment:
An alternative that might be worth looking into is the 'CURRENT_TIMESTAMP' keyword as default value in the table from "stardard SQL". I would assume most databases support this and then we don't have to do the time conversion. Although it's also not a big deal because this is only at account creation anyway.

Reply:
`CURRENT_TIMESTAMP` also seems to work as a macro:
```
sqlite> SELECT CURRENT_TIMESTAMP;
2023-10-22 06:19:46
```
So I think it could also just be used in an INSERT INTO statement. I'll have to check tho